spyder_kernels/console/tests/test_console_kernel.py

```diff
@@ -1080,7 +1065,7 @@ def test_locals_globals_in_pdb(kernel):
     pdb_obj.curframe_locals = None
 
 
-@flaky(max_runs=3)
+# @flaky(max_runs=3)
```

Comment:
Please remove this line if you don't consider it necessary anymore.

Reply:
I was testing because this test is flaky on python 3.8 and tk. I put it back